gen3-integration-tests/pages/login.py

```diff
@@ -54,7 +59,18 @@ def login(self, page: Page, user="main_account"):
 
     def logout(self, page: Page):
         """Logs out and wait for Login button on nav bar"""
-        page.get_by_role("button", name="Logout").click()
+        res = get_portal_config(pytest.namespace)
+        # Check if useProfileDropdown is set to True and perform logout accordingly
+        if (
+            "useProfileDropdown" in res["components"]["topBar"].keys()
+            and res["components"]["topBar"]["useProfileDropdown"]
+        ):
+            page.locator(self.USER_PROFILE_DROPDOWN).click()
+            page.wait_for_timeout(3000)
```

Comment:
we should avoid explicit waits, click has a 30 second implicit wait, we can increase that if needed